README.md

```diff
@@ -204,11 +204,14 @@ $beneficiary = $client->beneficiary()->externalAccount()
 ### 2. Creating a user
 
 ```php
+use TrueLayer\Constants\UserPoliticalExposures;
+
 $user = $client->user()
     ->name('Jane Doe')
     ->phone('+44123456789')
     ->email('jane.doe@truelayer.com')
-    ->dateOfBirth('2024-01-01');
+    ->dateOfBirth('2024-01-01')
+    ->politicalExposure(UserPoliticalExposures::NONE);
```

Comment:
Should we highlight this as optional? I think most people will not be using this property.